git-gui.sh

```diff
@@ -3272,7 +3272,7 @@ ttext $ui_workdir \
 	-borderwidth 0 \
 	-width 20 -height 10 \
 	-wrap none \
-	-takefocus 1 -highlightthickness 1\
+	-takefocus 1 -highlightthickness 1 -highlightcolor #f0f0f0 -highlightbackground #303030 \
```

Comment:
This is ugly; abstract it out. It doesn't work to use `$color::focused` because it might not have been initialized yet.